pyvrp/read.py

```diff
@@ -146,12 +151,15 @@ def read(
     if demands[0] != 0:
         raise ValueError("Demand of depot must be 0")
 
-    if time_windows[0, 0] != 0:
```

Comment:
I'm not sure why we require the depot time window to start at 0 in `read.py`. Idem for the other data. In Solution, we track the time starting at depot tw_early:

```
Duration time = data.depot().twEarly;
```

Reply:
I'm a bit late to this party, but yes, I think so. We should move what we really need to check into ProblemData's constructor. We might also want to lift some restrictions at that point.

Reply:
I have some initial work for this on the branch https://github.com/PyVRP/PyVRP/tree/non-zero-depot-early-tw. I'll finish it this weekend.